python/__init__.py

```diff
@@ -401,9 +406,19 @@ def is_uno(self):
 
   def is_dos(self):
     return self.get_type() == Panda.HW_TYPE_DOS
+  
+  def is_red(self):
+    return self.get_type() == Panda.HW_TYPE_RED_PANDA
+
+  def get_hw_generation(self):
```

Comment:
not needed I guess? maybe `has_canfd` might be better

Reply:
was following lib style, will change but this way it will be the only panda without it's own func )